lib/engine/game/g_18_co.rb

```diff
@@ -43,6 +43,16 @@ class G18CO < Base
       GREEN_TOWN_TILES = %w[co8 co9 co10].freeze
       BROWN_CITY_TILES = %w[co4 63].freeze
 
+      STOCKMARKET_COLORS = {
+        par: :yellow,
+        acquisition: :red,
+      }.freeze
+
+      MARKET_TEXT = {
+        par: 'Par: C [40, 50, 60, 75] - 40%, B/C [80, 90, 100, 110] - 50%, A/B/C: [120, 135, 145, 160] - 60%',
```

Comment:
Looks good, i'm putting a change in for 1867 that might make the par group a bit cleaner

Reply:
but that improvement can be done later.